app/views/RoomsListView/index.js

```diff
@@ -140,7 +142,9 @@ export default class RoomsListView extends LoggedView {
 		this.getSubscriptions();
 		const { navigation } = this.props;
 		navigation.setParams({
-			onPressItem: this._onPressItem, initSearchingAndroid: this.initSearchingAndroid, cancelSearchingAndroid: this.cancelSearchingAndroid
+			onPressItem: this._onPressItem,
```

Comment:
Unnecessary update

Reply:
With more one param this need object-curly-newline